ringcentral/assets/logs/ringcentral_tests.yaml

```diff
@@ -0,0 +1,259 @@
+            },
+            "reasonDescription": "The caller hung up before the call was answered.",
+            "lastModifiedTime": "2024-07-23T11:48:10.032Z",
+            "sessionId": "1234567890",
+            "internalType": "International",
+            "transport": "VoIP",
+            "type": "Voice",
+            "uri": "https://platform.ringcentral.com/restapi/v1.0/account/000000000/call-log/aKH2vDGxBbeYjUA?view=Detailed",
+            "billing": {
+              "costIncluded": 0,
+              "costPurchased": 0
+            },
+            "duration": 16,
+            "result": "Hang Up",
+            "telephonySessionId": "s-a7280be05beffz190df6b11d4z296c790000",
+            "legs": [
+              {
+                "reason": "Hang Up",
+                "extension": {
+                  "id": 123456789,
+                  "uri": "https://platform.ringcentral.com/restapi/v1.0/account/000000000/extension/000000000"
+                },
+                "reasonDescription": "The caller hung up before the call was answered.",
+                "legType": "SipToPstnUnmetered",
+                "internalType": "International",
+                "transport": "VoIP",
+                "type": "Voice",
+                "master": true,
+                "duration": 16,
+                "result": "Hang Up",
+                "telephonySessionId": "s-a7280be05beffz190df6b11d4z296c790000",
+                "action": "VoIP Call",
+                "startTime": "2024-07-23T12:19:34.664Z",
+                "from": {
+                  "phoneNumber": "+00000000000",
+                  "name": "xxxxxxxxxxx",
+                  "extensionId": "123456789",
+                  "device": {
+                    "id": "1234567890",
+                    "uri": "https://platform.ringcentral.com/restapi/v1.0/account/000000000/device/000000000"
+                  }
+                },
+                "to": {
+                  "phoneNumber": "+0000000000"
+                },
+                "partyId": "p-a7280be05beffz190df6b11d4z296c790000-2",
+                "durationMs": 1687,
+                "direction": "Outbound"
+              }
+            ],
+            "action": "VoIP Call",
+            "startTime": "2024-08-22T17:21:34.668Z",
+            "from": {
+              "phoneNumber": "+000000000",
+              "name": "xxxxxxx",
+              "extensionId": "00000000",
+              "device": {
+                "id": "0000000000",
+                "uri": "https://platform.ringcentral.com/restapi/v1.0/account/0000000/device/00000000"
+              }
+            },
+            "id": "000000000",
+            "to": {
+              "phoneNumber": "+0000000000"
+            },
+            "partyId": "p-a7280be05beffz190df6b11d4z296c790000-2",
+            "durationMs": 1607,
+            "direction": "Outbound"
+          },
+          "service": "voice",
+          "extensionId": 00000000
```

Comment:
This should either be a digit (`0`) or a string (`"00000000"`)

Reply:
We are receiving `extension.id` as an integer in the API response, so the pipeline will keep this attribute in integer format.